osama-breakout-room/qays/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  box-shadow: 0px 0px 5px 0px grey;
+  padding: 6px;
+}
+
+.wrapper {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.logo {
+  float: left;
+}
+
+.logo img {
+  width: 50px;
+  height: auto;
+}
+
+nav {
+  float: right;
+}
+
+.btn {
+  display: inline-block;
+  border-radius: 12px;
+  padding: 6px 20px;
+  margin: 6px;
+  text-align: center;
+  font-size: 14px;
+
+}
+
+a.btn {
+  text-decoration: none;
+}
+
+.btn-bordered {
+  border: 1px solid #1da1f2;
+  color: #1da1f2;
+}
+
+.btn-blue {
+  border: 1px solid #1da1f2;
+  background-color: #1da1f2;
+  color: white;
+}
+
+.profile-cover-container {
+  width: 100%;
+  height: 300px;
+}
+
+.profile-image-container {
+  width: 140px;
+  height: 140px;
+  overflow: hidden;
+  margin: -75px 0 -75px 50px;
+}
+
+.profile-image-container img {
+  width: 100%;
+  border-radius: 50%;
+  border: 2px solid white;
+}
```

Comment:
Big props on taking this approach! 👏 It is best practice to make your media to be fluid within the constraints of their parent container which will allow them to shrink and expand based on the window size without breaking your page.